### ISSUE TITLE
Unstable issue with new page added

### ISSUE THREAD

**Issue by underr-ua** [jimblue/grav-plugin-minify-html#12]
After added a new page i catched the bug -  unknown $() function. After some minutes of research i found that  page was minified incorrectly.

Page starts with tat code:
```
</script><script>
window.GoogleAnalyticsObject = 'ga';
window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
ga('create', 'UA-000000000-0', 'auto');
ga('set', 'forceSSL', true);
ga('send', 'pageview');
$(document).ready(function(){
    $('a[rel="lightbox"]').featherlightGallery({
        openSpeed: 250,
        closeSpeed: 250,
        closeOnClick: 'background',
        closeOnEsc: '1',
        root: 'body'
    });
});

</script></head><body class="...
```

**Comment by jimblue** [comment]
Hi @underr-ua and thanks for reporting this.

Could you create a gist with the complete source code of the page?
Your exemple is a bit short to understand what's going on.

Cheers

**Comment by underr-ua** [comment]
It happens when i add new page:
[minified](https://gist.github.com/underr-ua/8cea3ffc2ffe6b42668d5b844a4a1a15)
After disable the plugin:
[normal](https://gist.github.com/underr-ua/b9ae47698fb7f59d7898618cd2e6fcc3)

**Comment by jimblue** [comment]
ok @underr-ua, could you please make this two tests :
- remove google analytics script
- disable Grav JS and CSS minification

**Comment by underr-ua** [comment]
Yes i can, but this error is unstable. I reproduced it few times in the row and could not in the next 10 times. I willl try more later.

**Comment by jimblue** [comment]
Hey @underr-ua !

Is this fix with last release?

**Comment by underr-ua** [comment]
Hi! I temporarily disabled the plugin. I will try it the next few days and make report.

**Comment by jimblue** [comment]
Hi @underr-ua ? Can I close this issue?

**Comment by underr-ua** [comment]
Hi! I think yes - i could not repeat this behavior after recent update.

**Comment by underr-ua** [comment]
Hi @jimblue! Unfortunately i caught this bug again...

**Comment by jimblue** [comment]
Hi @underr-ua ! Are you sure it comes from `minify-html`!? I need more informations...

**Comment by jimblue** [comment]
Ok thanks, I will keep this close as you're the only one to report such issue and more informations are needed to reproduce it.

**Comment by underr-ua** [comment]
What information you need? 
Today i made some changes in a page and catch the bug again - after page saving it source was look like i wrote above in the [gist](https://gist.github.com/underr-ua/8cea3ffc2ffe6b42668d5b844a4a1a15). I tried to update the page (F5, Ctrl+F5), but nothing changed, then i turn off `minify-html` and refresh the page - it looked normal as i expected.

P.S.: Sorry if my English is too bad, i never studied it)

**Comment by jimblue** [comment]
I can't replicate your issue here. Can you please send me an archive of your project so I can understand what's going on?
